### DISCUSSION TITLE
Open A New Tab With Double Click On Vertical Tab Bar

### DISCUSSION THREAD

**Discussion by NeonEDuck** [zen-browser/desktop#1300] [Ideas]
### What feature would you like to see?
Ability to open new tab with double click on empty space in the vertical tab bar.
It would be a similar behavior like how you see in some code editor like VS Code or Notepad++.

**Comment by Bu11etmagnet** [comment]
Waterfox does the same. 
It's annoying that Zen does maximize/restore instead. There's a title bar for that. Oh sorry, the title bar has been taken over by the toolbar, the search bar and stuff.

**Comment by keiothic** [comment]
Yes! I would love this as well. Hopefully it's in there roadmap to include.  The fact that it's associated to a title bar double click action seems like a wasted opportunity of a vertical bar
